etna/articles/models.py

```diff
@@ -371,12 +379,19 @@ class Meta:
                 FieldPanel("custom_warning_text"),
             ],
         ),
-        InlinePanel(
-            "gallery_images",
+        MultiFieldPanel(
```

Comment:
Why is this changed from InlinePanel to MultiFieldPanel? Is this anywhere in the requirements?

Reply:
Hi Allan, it's not part of the requirements but just how we tend to lay things out in the CMS. I've chosen to do this as it allows the "Image Gallery" components to be grouped together nicely in the CMS 🙂

Reply:
Can you please make a mention in the ticket what this change is about, so that we can proceed.
_Re:  Ensured that PR includes only commits relevant to the ticket._

Reply:
And how to test this change as well.

Reply:
This is relevant to the ticket, it's part of the "process" I suppose 🙂 It just bunches the fields together so that they are included in a drop down - otherwise they are separate. Makes for a nicer editor experience, but is relevant as it's part of the adding of the `gallery_heading`